experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -658,6 +658,7 @@ const ChartTile: React.FC<{
           places={placeDcids.join(" ")}
           sort="descending"
           showExploreMore={true}
+          variableNameRegex={"(?<=\\[)(.*?)(?=\\])"}
```

Comment:
Let's extract this out to a constant since it's shared with the line chart?

Reply:
done